docs/src/content/en/docs/workflows-vnext/inngest-workflow.mdx

```diff
@@ -151,30 +96,25 @@ Register the workflow with Mastra and configure the Inngest API endpoint:
 ```ts showLineNumbers copy filename="src/mastra/index.ts"
 import { Mastra } from '@mastra/core/mastra'
 import { serve as inngestServe } from '@mastra/inngest'
+import { incrementWorkflow } from './workflows'
+import { inngest } from './inngest'
 import { PinoLogger } from '@mastra/loggers'
-import { Inngest } from 'inngest'
-import { incrementWorkflow } from './workflows/inngest-workflow'
-import { realtimeMiddleware } from '@inngest/realtime'
-import { serve } from '@hono/node-server'
-import { createHonoServer } from '@mastra/deployer/server'
-
 
-// Create an Inngest instance with realtime middleware for development
-const inngest = new Inngest({
-  id: 'mastra',
-  baseUrl: `http://localhost:8288`,
-  isDev: true,
-  middleware: [realtimeMiddleware()],
-})
 
 // Configure Mastra with the workflow and Inngest API endpoint
 export const mastra = new Mastra({
   vnext_workflows: {
     incrementWorkflow
   },
   server: {
+    // The server configuration is required to expose the Inngest integration endpoint.
```

Comment:
The reason it's required for local docker setup is so that the docker container (inngest dev server) can connect to the host port. The other stuff isn't all that important imo.